examples/dt/uni04delta/kustomization.yaml

```diff
@@ -9,3 +9,13 @@ resources:
   - control-plane/nncp/values.yaml
   - control-plane/service-values.yaml
   - values.yaml
+
+patches:
+  - target:
+      version: v1beta1
+      kind: OpenStackControlPlane
+      name: controlplane
+    patch: |-
+      - op: add
+        path: /spec/manila/template/manilaShares/share1/networkAttachments/-
+        value: nfs
```

Comment:
Not necessary for now; as discussed [elsewhere](https://github.com/openstack-k8s-operators/architecture/pull/376/files#r1751682609). May need to revisit this in the future if we decide to support data migrations facilitated by a control plane service.

Reply:
thanks. removed.